examples/documentation/documentation.bal

```diff
@@ -5,47 +5,41 @@ function main(string... args) {
 }
 
 // This is the documentation attachment for object `DummyObject`. You may use Markdown syntax to style the text in these descriptions.
```

Comment:
Do we need this test? "You may use Markdown syntax to style the text in these descriptions."